### PR TITLE
Patch LP_I2C for C6

### PR DESCRIPTION
**Pull request by JurajSadel**

### CODE REVIEW

**Review by bjoernQ — APPROVED** [review]
LGTM

File: esp32c6-lp/svd/patches/esp32c6-lp.yaml [hunk 1]
```diff
@@ -1 +1,10 @@
 _svd: "../esp32c6-lp.base.svd"
+
+LP_I2C0:
+  _strip: "I2C_"
+  "*":
+    _strip: "I2C_"
+  DATA:
+    _modify:
+      FIFO_RDATA:
+        access: read-write
```
Comment (MabezDev) [inline esp32c6-lp/svd/patches/esp32c6-lp.yaml:10]: Is FIFO rddata actually read-write? Or is it readonly?

**Review by MabezDev — CHANGES_REQUESTED** [review]

File: esp32c6-lp/svd/patches/esp32c6-lp.yaml (hunk 1 above)
Comment (JurajSadel) [reply esp32c6-lp/svd/patches/esp32c6-lp.yaml:10]: It's read-write, we are using it in HP i2c as well: https://github.com/esp-rs/esp-hal/blob/main/esp-hal/src/i2c.rs#L1555-L1558
The name is very confusing :D

Comment (MabezDev) [reply esp32c6-lp/svd/patches/esp32c6-lp.yaml:10]: Haha thanks!

**Review by MabezDev — APPROVED** [review]

**Pull request merged**